### PR TITLE
#54 - Added blob HEAD route

### PR DESCRIPTION
**Pull request by olegmoz**
Part of #54 
PullLayer transformed to Blob.Get(), added Blob.Head() route

### CODE REVIEW

**Comment by olegmoz** [comment]
@g4s8 please take a look at this issue, no reviewer has been assigned for a long time

File: src/main/java/com/artipie/docker/http/BlobEntity.java [hunk 1]
```diff
@@ -39,58 +39,92 @@
 import org.reactivestreams.Publisher;
 
 /**
- * Slice for pull image manifest endpoint.
- * See <a href="https://docs.docker.com/registry/spec/api/#pulling-an-image">Pulling An Image</a>.
+ * Blob entity in Docker HTTP API.
+ * See <a href="Blob">Blob</a>.
```
Comment (g4s8) [inline src/main/java/com/artipie/docker/http/BlobEntity.java:43]: @olegmoz it seems `href` is broken here

File: src/main/java/com/artipie/docker/http/BlobEntity.java [hunk 2]
```diff
@@ -39,58 +39,92 @@
...
+     */
+    static final class Head implements Slice {
+
+        @Override
+        public Response response(
+            final String line,
+            final Iterable<Map.Entry<String, String>> headers,
+            final Publisher<ByteBuffer> body
+        ) {
+            return new RsWithStatus(RsStatus.OK);
```
Comment (g4s8) [inline src/main/java/com/artipie/docker/http/BlobEntity.java:127]: @olegmoz maybe 404 code should be here if blob was not found?

Comment (olegmoz) [reply src/main/java/com/artipie/docker/http/BlobEntity.java:127]: @g4s8 indeed it should. But right now we are missing method to check that blob exists without reading it's content. I will get to it later

File: src/main/java/com/artipie/docker/http/BlobEntity.java (hunk 1 above)
Comment (olegmoz) [reply src/main/java/com/artipie/docker/http/BlobEntity.java]: @g4s8 fixed, thank you for noticing

**Pull request merged**